<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportByClient</class>
 <widget class="QWidget" name="ReportByClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1285</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="leftMargin">
      <number>45</number>
     </property>
     <property name="rightMargin">
      <number>45</number>
     </property>
     <item>
      <widget class="QPushButton" name="BackButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>52</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>НАЗАД</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="EcoTaxi">
         <property name="text">
          <string>ОТЧЁТ</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Header">
         <property name="text">
          <string>По Должнику</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="SettingsButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>52</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>НАСТРОЙКИ</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,2">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>30</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="sideTable">
         <property name="maximumSize">
          <size>
           <width>280</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="minimumSize">
        <size>
         <width>700</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777210</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="tableView">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>300</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="bottomTable">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="totalTable">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>75</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>698</number>
     </property>
     <property name="rightMargin">
      <number>45</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ToEXCELButton">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>52</height>
        </size>
       </property>
       <property name="text">
        <string>ЭКСПОРТ В EXCEL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ToPDFButton">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>52</height>
        </size>
       </property>
       <property name="text">
        <string>ЭКСПОРТ  В PDF</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
